src/generator/AutoRest.Go/rakefile.rb

```diff
@@ -196,8 +200,9 @@ def to_services(m, h, *p)
 def generate(service)
 	s = "Generating #{service.fullname}.#{service.version} "
 	puts "#{s} #{"=" * (80 - s.length)}"
+
 	delete(service)
-	s = `#{AUTOREST} -AddCredentials -CodeGenerator Go -Header MICROSOFT_APACHE -Input #{service.input_path} -Namespace #{service.namespace} -OutputDirectory #{service.output_path} -Modeler Swagger`
+	s = `#{AUTOREST} -AddCredentials -CodeGenerator Go -Header MICROSOFT_APACHE -Input #{service.input_path} -Namespace #{service.namespace} -OutputDirectory #{service.output_path} -Modeler Swagger -pv #{ENV['VERSION']}`
```

Comment:
Can you use `SDK_VERSION` variable in the command instead of accessing environment variable again?